src/System.Text.RegularExpressions/tests/RegexMatchTests11.cs

```diff
@@ -9,219 +9,84 @@
 public partial class RegexMatchTests
 {
     /*
-    Tested Methods:
-
-        public static Match Match(string input);     Numbering pattern slots: "(?< 1>\\d{3})(?< 2>\\d{3})(?< 3>\\d{4})"
-            "8885551111"
-
-        public static Match Match(string input);     Not naming pattern slots at all: "^(cat|chat)"
-            "cats are bad"
-
-        public static Match Match(string input);      "([0-9]+(\\.[0-9]+){3})"
-            "209.25.0.111"
-
+    public Match Match(string input);
+        - Numbering pattern slots: "(?< 1>\\d{3})(?< 2>\\d{3})(?< 3>\\d{4})", "8885551111"
+        - Not naming pattern slots at all: "^(cat|chat)", "cats are bad"
+        - "([0-9]+(\\.[0-9]+){3})", "209.25.0.111"
     */
-
     [Fact]
     public static void RegexMatchTestCase11()
```

Comment:
Same naming question... I'll stop commenting on these... would seem to apply to all of the RegexMatchTests#.cs files